src/main/java/project/controllers/CreateCarsController.java

```diff
@@ -0,0 +1,38 @@
+package project.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import project.lib.Injector;
+import project.models.Car;
+import project.models.Manufacturer;
+import project.service.CarService;
+import project.service.ManufacturerService;
+
+public class CreateCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("project");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final ManufacturerService manufacturerService = (ManufacturerService)
+            injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/createCars.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/car/create.jsp").forward(req, resp);
```
I would recommend to put JSPs (and controllers) to different folders